src/index.html

```diff
@@ -11,7 +11,198 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      autocomplete="off"
+    >
+      <fieldset class="fieldsets">
+        <legend>Personal information</legend>
+
+        <label class="form-field">
+        Surname:
+          <input
+              type="text"
+              name="surname"
+              required
+          >
```

Comment:
https://www.kindacode.com/article/vs-code-how-to-change-indentation-2-spaces-4-spaces/

fix in all cases

```suggestion
          <input
            type="text"
            name="surname"
            required
          >
```